core/templates/dev/head/css/oppia.css

```diff
@@ -1026,8 +1089,8 @@ md-card.oppia-gallery-tile {
 }
 
 .oppia-html-select .oppia-html-select-option oppia-noninteractive-image {
```

Comment:
I do not feel comfortable reinstating the changes. As I did not make the changes, I am not sure which changes to keep and which to changes to revert.